website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc

```diff
@@ -0,0 +1,156 @@
+** link:https://freebsdfoundation.org/blog/july-2023-software-development-projects-update/[July 2023 Software Development Projects Update]
+** https://freebsdfoundation.org/blog/freebsd-for-research-cheri-morello/[FreeBSD for Research: CHERI/Morello]
+** Meet the FreeBSD Google Summer of Code Students
+*** link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-soobin-rho/[Soobin Rho]
+*** link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-raghav-sharma/[Raghav Sharma]
+*** link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-sudhanshu-mohan-kashyap/[Sudhanshu Mohan Kashyap]
+*** link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-aymeric-wibo/[Aymeric Wibo]
+** link:https://freebsdfoundation.org/blog/meet-the-summer-2023-university-of-waterloo-co-op-student-naman-sood/[Meet The Summer 2023 University of Waterloo Co-Op Student: Naman Sood]
+** link:https://freebsdfoundation.org/blog/meet-freebsd-foundation-2023-summer-intern-jake-freeland/[Meet FreeBSD Foundation 2023 Summer Intern: Jake Freeland]
+* FreeBSD in the News:
+** link:https://freebsdfoundation.org/news-and-events/latest-news/freebsd-foundation-joins-osis-open-policy-alliance/[FreeBSD Foundation Joins OSI's Open Policy Alliance]
+** link:https://freebsdfoundation.org/news-and-events/latest-news/hackernoon-5-reasons-we-use-open-source-freebsd-as-our-enterprise-os/[Hackernoon: 5 Reasons We Use Open Source FreeBSD as Our Enterprise OS]
+** link:https://freebsdfoundation.org/news-and-events/latest-news/what-the-dev-podcast-the-evolution-of-the-freebsd-project/[What the Dev Podcast: The Evolution of the FreeBSD Project]
+
+We help educate the world about FreeBSD by publishing the professionally produced FreeBSD Journal.
+As we mentioned previously, the FreeBSD Journal is now a free publication.
+Find out more and access the latest issues at link:https://www.freebsdfoundation.org/journal/[].
+
+You can find out more about events we attended and upcoming events at link:https://www.FreeBSDfoundation.org/news-and-events/[].
+
+==== OS Improvements
+
+During the third quarter of 2023, 282 src, 652 ports, and 24 doc tree commits identified The FreeBSD Foundation as a sponsor.
+Some of this Foundation-sponsored work is described in separate report entries:
+
+* <<_enabling_snapshots_on_filesystems_using_journaled_soft_updates,Enabling Snapshots on Filesystems Using Journaled Soft Updates>>
+* <<_login_classes_fixes_and_improvements,Login Classes Fixes and Improvements>>
+* <<_openssl_3_in_base_-_improved,OpenSSL 3 in base - improved>>
+* <<_openstack_on_freebsd,OpenStack on FreeBSD>>
+* <<_process_visibility_security_policies,Process Visibility Security Policies>>
+* <<_simd enhancements for amd64,SIMD enhancements for amd64>>
+
```

Comment:
Lorenzo, I assume that these lines will be edited during collation, for links to work.

Reply:
I will check them and fix them if necessary, but this is an adoc syntax that should work. See also the foundation's report from last quarter.